src/Facebook/InstantArticles/Transformer/Rules/ImageRuleParagraph.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+namespace Facebook\InstantArticles\Transformer\Rules;
+
+use Facebook\InstantArticles\Elements\Image;
+use Facebook\InstantArticles\Elements\TextContainer;
+use Facebook\InstantArticles\Elements\InstantArticle;
+use Facebook\InstantArticles\Transformer\Warnings\InvalidSelector;
+
+class ImageRuleParagraph extends ConfigurationSelectorRule
```

Comment:
I think the naming standard is that all rule classes ends with Rule. What about `ImageInsideParagraphRule`?